sandbox/.ipynb_checkpoints/HOWTO_PlotData-checkpoint.ipynb

```diff
@@ -39,8 +56,8 @@
     "    ls_ct = dff['control_data']\n",
     "    data = np.zeros((len(ls_wv),2))\n",
     "    for idx in range(len(ls_wv)):\n",
-    "        data[idx,0] = ls_wv[idx][4]  # time\n",
-    "        data[idx,1] = ls_wv[idx][3]  # pressure\n",
+    "        data[idx,0] = ls_wv[idx][6]  # time\n",
+    "        data[idx,1] = ls_wv[idx][5]  # pressure\n",
```

Comment:
This was to reflect the changes to the logging.

Reply:
Oh, these are dictionaries; can just be done with `data[:,0] = ls_wv['time']`etc. without the loop. Should have updates this as welel